test/auro-icon.test.js

```diff
@@ -70,6 +71,17 @@ describe('auro-icon', () => {
     expect(svg).to.not.be.null;
   });
 
+  it ('does not duplicate requests for same icon source', async () => {
```

Comment:
Oops, removing extra space between test function and opening paren.

```suggestion
  it('does not duplicate requests for same icon source', async () => {
```